common/lib/capa/capa/responsetypes.py

```diff
@@ -1240,12 +1240,14 @@ def mc_setup_response(self):
                     i += 1
                 # If using the masked name, e.g. mask_0, save the regular name
                 # to support unmasking later (for the logs).
-                if self.has_mask():
-                    mask_name = "mask_" + str(mask_ids.pop())
-                    self._mask_dict[mask_name] = name
-                    choice.set("name", mask_name)
-                else:
-                    choice.set("name", name)
+                # Masking is currently disabled so this code is temporarily commented, as
```

Comment:
Define: temporarily. I worry about this sitting unchanged forever and it never being revisited because "I dunno; @sarina did that right before she left." :(